lib/search.rb

```diff
@@ -265,6 +265,19 @@ def self.advanced_filters
     @advanced_filters
   end
 
+  advanced_filter(/^in:tagged$/) do |posts|
+    posts
+      .joins("JOIN topic_tags ON
```

Comment:
wait I think this expands the result set ... don't we want?

.where('EXISTS (SELECT 1 FROM topic_tags WHERE topic_tags.topic_id = posts.topic_id)')

Reply:
I'm unfamiliar with the result set semantics and was mostly just copying what I saw for similar tags (in:seen, in:unseen). Happy to try with a subselect both here and for in:untagged _a la_
```
.where(`NOT EXISTS...`)
```

Reply:
Tested locally in the UI & pushed.